shared/tracker/index.js

```diff
@@ -144,7 +93,18 @@ export default connect(
   }),
   dispatch => {
     return bindActionCreators(trackerActions, dispatch)
-  })(Tracker)
+  },
+  (stateProps, dispatchProps, ownProps) => ({
+    ...ownProps,
+    ...stateProps,
+    ...dispatchProps,
+    onClose: () => dispatchProps.onClose(ownProps.username),
```

Comment:
Ohh I see, a merge that maps over these functions wrapping them in the `ownProps.username` closure pattern?